apis/metacontroller/v1alpha1/types.go

```diff
@@ -130,6 +133,9 @@ type CompositeControllerList struct {
 // +genclient:noStatus
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=controllerrevisions,scope=Namespaced
+
```

Comment:
remove this blank like, on `make generated_files` it cause codegen ignores `ControllerRevision` type

Reply:
Sure thing, done.

Reply:
I can  still see line 138 is blank

Reply:
Ah, the change wasn't staged. The blank line should be gone now.